src/InputStepper/__tests__/index.test.js

```diff
@@ -42,6 +42,8 @@ describe(`InputStepper with help, prefix and suffix`, () => {
       onFocus={onFocus}
       onBlur={onBlur}
       spaceAfter={spaceAfter}
+      titleIncrement="Add"
```

Comment:
Can we put this into translations? I would use this properties for overrides. `Add` and `Remove` is totally okay for 90 percent of usages.

Reply:
@vepor we are not providing a default, so it is really needed?

Reply:
That's the thing, I think that we should - that accessibility is part of all components and when someone needs some different labels etc. there is a way